AUTHORS

```diff
@@ -21,3 +21,5 @@ Chronological list of authors
 
  2018
   - Shujie Fan
+  - Richard J Gowers
```

Comment:
I think you also have to add yourself to `docs/conf.py` – we haven't introduced fancy authorship processing here.